hedera-node/hedera-app/src/main/java/com/hedera/node/app/bbm/singleton/StakingInfo.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.bbm.singleton;
```

Comment:
This class `StakingInfo.java` already is a generated class. We can rename this class to something else that states its used only in this migration.

Reply:
I totally agree. 
But in all dumpers we use this convention (record with similar name of the corresponding state).
For example `com.hedera.node.app.bbm.files.FileId`, `com.hedera.node.app.bbm.nfts.UniqueToken`, ect...
So I'm not really sure what name should I use here.

FYI this PR will be merged in #11275